rust/src/builtins/apply_live.rs

```diff
@@ -97,8 +97,12 @@ fn finish(sysroot: &ostree::Sysroot) -> Result<()> {
     };
     pkgdiff.print();
 
-    crate::ffi::output_message(
-        "Successfully updated running filesystem tree; some services may need to be restarted.",
-    );
+    if !opts.allow_replacement {
```

Comment:
Shouldn't this be more something like `if pkgdiff.n_removed() == 0 && pkgdiff.n_modified() == 0 {` ?

Reply:
Yep, definitely better!